lighty-examples/lighty-community-restconf-dom-actions-app/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,4 @@
+log4j.rootLogger=INFO, STDOUT
```

Comment:
log4j config should not be needed
the config is inherited from `log4j-properties` dependency

Reply:
removed